src/init.cpp

```diff
@@ -447,11 +447,13 @@ void SetupServerArgs()
 
 #if ENABLE_ZMQ
     gArgs.AddArg("-zmqpubhashblock=<address>", "Enable publish hash block in <address>", false, OptionsCategory::ZMQ);
+    gArgs.AddArg("-zmqpubhashblockheader=<address>", "Enable publish hash block in <address>", false, OptionsCategory::ZMQ);
```

Comment:
The comment wasn't changed. And the name implies that you are sending the entire block header -- more than just hash and height.